lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelBatchHydrationByObjectId.kt

```diff
@@ -130,17 +129,12 @@ internal object NadelBatchHydrationByObjectId {
     private fun getHydrateInstructionsForNodeMatchingObjectId(
         executionBlueprint: NadelOverallExecutionBlueprint,
         state: NadelBatchHydrationTransform.State,
+        instruction: NadelBatchHydrationFieldInstruction,
         sourceNode: JsonNode,
         sourceIds: List<List<Any?>>,
         resultNodesByObjectId: Map<List<Any?>, JsonMap>,
     ): NadelResultInstruction {
-        val hydratedFieldDef = NadelTransformUtil.getOverallFieldDef(
```

Comment:
This is redundant, it looks up the field definition using the `__typename` in the result, but that's already been done because we've resolved an instruction at the call site.

This change just passes in the instruction and looks an the field definition in the instruction.

Likely this was a mistake due to copy paste during refactoring.